<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hp_SettingsDlg</class>
 <widget class="QDialog" name="hp_SettingsDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>455</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hpSettings">
      <attribute name="title">
       <string>Home Settings</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>20</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Angle Measure:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_angle">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Radians</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Radians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Degrees</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gradians</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="ch_signed">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>150</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Â±:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_format">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Standard</string>
       </property>
       <item>
        <property name="text">
         <string>Standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fixed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scientific</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Floating</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rounded</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cb_grouping">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>81</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>123,456.789</string>
       </property>
       <item>
        <property name="text">
         <string>123,456.789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123'456.789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123 456.789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123456.789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123.456,789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123'456,789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123 456,789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123&quot;456.789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123456,789</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1234,5678.901</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1,23,456.789</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Number Format:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Digit Grouping:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_entry">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>112</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Textbook</string>
       </property>
       <item>
        <property name="text">
         <string>Textbook</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Algebraic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RPN</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>111</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Entry:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_integers">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>144</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Binary</string>
       </property>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Octal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Decimal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hex</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>143</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Integers:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>175</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Complex:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_complex">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>176</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>a+b*i</string>
       </property>
       <item>
        <property name="text">
         <string>a+b*i</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(a,b)</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cb_language">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>208</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Deutsch</string>
       </property>
       <item>
        <property name="text">
         <string>Deutsch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Espanol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Francais</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nederlands</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Portugues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>207</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Language:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>239</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Font Size:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_font">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>240</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Small Font</string>
       </property>
       <item>
        <property name="text">
         <string>Small Font</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium Font</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large Font</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="ch_textbook">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>277</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Textbook Display:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ch_menu">
       <property name="geometry">
        <rect>
         <x>151</x>
         <y>277</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Menu Display:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_theme">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>311</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Light</string>
       </property>
       <item>
        <property name="text">
         <string>Light</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Color Theme:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_precision">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>51</width>
         <height>32</height>
        </rect>
       </property>
       <property name="currentText">
        <string>0</string>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="le_integer">
       <property name="geometry">
        <rect>
         <x>266</x>
         <y>144</y>
         <width>41</width>
         <height>32</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>32</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>hp_SettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>hp_SettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
